pyproject.toml

```diff
@@ -7,27 +7,6 @@ version_files = [
   ".pre-commit-config.yaml:rev:.+Commitizen"
 ]
 
-[tool.black]
```

Comment:
Please correct me if I'm wrong, we're removing these lines because they're default settings right?

Reply:
Yes, none of the lines removed affected anything. In many other projects I haven't configured anything for black and it just works. I thought it's nice to do some clean up, and only add if we have to